nuxeo-services/login/nuxeo-platform-login-openid/src/main/java/org/nuxeo/ecm/platform/oauth2/openid/OpenIDConnectProviderDescriptor.java

```diff
@@ -89,6 +104,12 @@
 
     @XNode("userInfoClass")
     protected Class<? extends OpenIDUserInfo> userInfoClass = DEFAULT_USER_INFO_CLASS;
+    
+    /**
+     * @since 11.1
+     */
+    @XNode("userInfoAuth")
+    protected String userInfoAuth = DEFAULT_USER_INFO_AUTH;
```

Comment:
I suggest renaming this to `authenticationMethod` everywhere, as I think it's clearer.

Reply:
Done